src/routes/docs/tutorials/refine/step-3/+page.markdoc

```diff
@@ -51,7 +51,7 @@ const APPWRITE_PROJECT = "<YOUR_PROJECT_ID>"; // Replace with your project ID
 
 const appwriteClient = new Appwrite();
 
-appwriteClient.setEndpoint(APPWRITE_URL).setProject(APPWRITE_PROJECT);
```

Comment:
Other parts of the tutorial needs to be updated too since this is removed, right?